components/FileFormats.vue

```diff
@@ -16,7 +16,7 @@
 			:collapsed="collapsed"
 			@detailsToggled="detailsToggled">
 			<template #heading="scope"><slot name="heading" v-bind="scope" /></template>
-			<template v-if="missing" #content-start><FederationMissingNotice :missing="missing" :federation="federation" /></template>
+			<template v-if="missing && Array.isArray(missing) && missing.length>0" #content-start><FederationMissingNotice :missing="missing" :federation="federation" /></template>
```

Comment:
```suggestion
			<template v-if="Array.isArray(missing) && missing.length>0" #content-start><FederationMissingNotice :missing="missing" :federation="federation" /></template>
```

see https://github.com/Open-EO/openeo-vue-components/pull/102/files#r2036266968